docs/proposals-accepted/202206-active-series-limiting-hashring.md

```diff
@@ -0,0 +1,207 @@
+---
+type: proposal
+title: Active Series Limiting for Hashring Topology
+status: accepted
+owner: saswatamcode
+menu: proposals-accepted
+---
+
+## Related links/tickets
+
+* https://github.com/thanos-io/thanos/pull/5333
+* https://github.com/thanos-io/thanos/pull/5402
+* https://github.com/thanos-io/thanos/issues/5404
+* https://github.com/thanos-io/thanos/issues/4972
+
+## Why
+
+Thanos is built to be a set of components that can be composed into a highly available metrics system with unlimited storage capacity. But for achieving true HA we need to ensure that tenants in our write path, cannot push too much data and cause issues. There need to be limits in place so that our ingestion systems maintain the level of QoS and only block the tenant that exceeds limits.
```

Comment:
```suggestion
Thanos is built to be a set of components that can be composed into a highly available metrics system with unlimited storage capacity. But for achieving true HA we need to ensure that tenants in our write path, cannot push too much data and cause issues. There need to be limits in place so that our ingestion systems maintain the level of  Quality of Service (QoS) and only block the tenant that exceeds limits.
```

Sorry, my book writing intuition comes in 🙈